com.unity.shadergraph/Editor/Generation/TargetResources/Fields.cs

```diff
@@ -10,26 +10,16 @@ internal static class Fields
 #endregion
 
 #region Fields
+        // These are core Fields shared between URP and HDRP etc.
         public static FieldDescriptor GraphVertex =           new FieldDescriptor(kFeatures, "graphVertex", "FEATURES_GRAPH_VERTEX");
         public static FieldDescriptor GraphPixel =            new FieldDescriptor(kFeatures, "graphPixel", "FEATURES_GRAPH_PIXEL");
         public static FieldDescriptor AlphaClip =             new FieldDescriptor(string.Empty, "AlphaClip", "_AlphaClip 1");
         public static FieldDescriptor AlphaTest =             new FieldDescriptor(string.Empty, "AlphaTest", "_ALPHA_TEST 1");
-        public static FieldDescriptor SurfaceOpaque =         new FieldDescriptor(kSurfaceType, "Opaque", "_SURFACE_TYPE_OPAQUE 1");
-        public static FieldDescriptor SurfaceTransparent =    new FieldDescriptor(kSurfaceType, "Transparent", "_SURFACE_TYPE_TRANSPARENT 1");
-        public static FieldDescriptor BlendAdd =              new FieldDescriptor(kBlendMode, "Add", "_BLENDMODE_ADD 1");
```

Comment:
Why are we removing this fields and adding UniversalFields, it seems most of them are generic/pipeline agnostic.

Reply:
I see the comment: "Moved universal only Fields from "shared" Fields.cs to UniversalFields.cs for clearer visibility what is actually shared. We need to revisit this and define "Core" fields." 

I don't have a strong opinion about it as this is all internal API, but it seems to me these fields are generic enough, if HDRP is duplicating them it should be HDRP that stop duplication and use the ones in shared Fields type.

Reply:
No they aren't. They're actually Universal only. f.ex. HDRP has it's own definitions for these. 
The intent is to build back the "core" with things that are actually shared. This is just a step to untangle this for now.